### DISCUSSION TITLE
RTX 3060 12GB + other

### DISCUSSION THREAD

**Discussion by stejpet** [MrNeRF/LichtFeld-Studio#48] [General]
Just wanted to inform that RTX 3060 12GB works fine under linux.

Ubuntu 23.04, Kernel 6.3.7-060307-generic, nvidia-driver 535.

Everything works fine, and was easy to use. However, I wanted to know it is correct that the RAM usage using original images (1917*1076) is above 32GB, but halved images is only 8GB RAM. I have to use resized images because I only have 32GB, so OOM :(

PS: Any thoughts about supporting HIP?

**Comment by MrNeRF** [comment]
Hi, thanks for letting me know!
If you have width = N /2 and height = M/2 your mem consumption will be then  N/2 * M/2, i.e. down by a factor 4 => 8GB sounds reasonable.
What is HIP?

**Reply by stejpet** [reply]
According to AMD: HIP is a C++ runtime API and kernel language that allows developers to create portable applications for AMD and NVIDIA GPUs from single source code.

There is a tool, or several tools to convert cuda code to HIP. I haven't tried it, but it could be worth taking a look maybe.
https://rocm.docs.amd.com/projects/HIP/en/latest/

**Comment by MrNeRF** [comment]
Good to know. But that's beyond I have currently time to do. If you would like to try it out, just go for it.
